data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -223,6 +223,24 @@
     "Michigan EJSCREEN Priority Community"
 )
 
+# CDC SVI INDEX percentile fields
+CDC_SVI_INDEX_SE_THEME_FIELD: str = "Socioeconomic Index"
```

Comment:
Nit: maybe for clarity (as previously discussed) as "SVI " to the beginning of all these names? E.g., put yourselves in the shoes of a developer who is opening the `usa.csv` which has 300+ columns in it and they're trying to figure out where data came from.

Reply:
Yes, I will do so